packages/test/snapshots/src/manifest.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fs from "fs";
+
+/**
+ * The current version of the snapshot tests. This should be updated if there is a change in how the tests validates
+ * snapshot. For example, there was a change to generate snapshots not at fixed points (after 1000 ops) but at the same
+ * points (sequence numbers) as what happened in the original file. The version was upgraded because snapshots added
+ * before and after that point ran different validations.
+ */
+export const currentTestVersion: number = 2;
+/**
+ * The version of the test after which "testSummaries" replay args is set, indicating that snapshots need to be
+ * generated at the same points as the original file did.
+ */
+export const testSummariesVersion: number = 2;
+
+interface ITestManifest {
+	/** The version of the snapshot tests when a particular test folder was added. */
+	testVersion: number;
```

Comment:
An alternative is to add features here that are added. For example, SharedMatrixDeletes and newer snapshots will have "testSummaries" property set in manifest.